jhub_apps/themes/__init__.py

```diff
@@ -1,7 +1,9 @@
 LOGO = "/services/japps/static/img/Nebari-Logo-Horizontal-Lockup-White-text.svg"
+FAVICON = "/services/japps/static/favicon.ico"
 
 DEFAULT_THEME = {
```

Comment:
I think, we can add version here so that it doesn't defaults to `v0.0.1`, but not blocking for this this, we can do it in a separate PR.